src/list/components/item-fields.js

```diff
@@ -9,35 +9,51 @@ import React from "react";
 import CopyToClipboardButton from "../../widgets/copy-to-clipboard-button";
 import FieldText from "../../widgets/field-text";
 import Input from "../../widgets/input";
+import Button from "../../widgets/button";
 import LabelText from "../../widgets/label-text";
 import PasswordInput from "../../widgets/password-input";
 import PasswordText from "../../widgets/password-text";
 
 import styles from "./item-fields.css";
 
 const fieldsPropTypes = PropTypes.shape({
+  isPopup: PropTypes.bool,
   origin: PropTypes.string.isRequired,
   title: PropTypes.string.isRequired,
   username: PropTypes.string.isRequired,
   password: PropTypes.string.isRequired,
 });
 
-export function ItemFields({fields, onCopy}) {
+export function ItemFields({fields, onCopy, isPopup}) {
+  const originEl = isPopup ? (
+      <h4 className={styles.popupOrigin}>{new URL(fields.origin).host}</h4>
+  ) : (
+      <a className={styles.originLink} href={fields.origin}>{fields.origin}</a>
```

Comment:
Sorry for not catching this earlier!  The link should behave the same as the "Launch" button; open the site in a new tab/window (see comment below for details).